app/views/shared/_sidebar_navigation.html.erb

```diff
@@ -1,13 +1,13 @@
-<% if local_assigns.has_key?(:tasklist) && show_tasklist_sidebar? %>
-  <%= render 'shared/sidebar_tasklist', tasklist: tasklist %>
-<% elsif @navigation.should_present_taxonomy_navigation? -%>
-  <%# FIXME: Markup can wrap all three sidebar navigation
-      once we've migrated the appropriate formats %>
-  <div class="column-third add-title-margin">
+<div class="column-third">
+  <% if local_assigns.has_key?(:tasklist) -%>
+    <% tasklist = local_assigns[:tasklist] -%>
+    <h2 class="task-list-title">
+       <a href="<%= tasklist[:base_path] %>"><%= tasklist[:title] %></a>
+    </h2>
+    <%= render 'govuk_component/task_list', tasklist[:tasklist] %>
+  <% elsif @navigation.should_present_taxonomy_navigation? -%>
     <%= render partial: 'govuk_component/taxonomy_sidebar', locals: @content_item.taxonomy_sidebar %>
```

Comment:
When pulling in components, you don't need "partial:" or "locals:" before each. Same for the line below.